rustls/src/conn.rs

```diff
@@ -343,7 +343,9 @@ impl ConnectionSecrets {
         ret
     }
 
-    pub fn make_key_block(&self, len: usize) -> Vec<u8> {
+    pub fn make_key_block(&self, scs: &SupportedCipherSuite) -> Vec<u8> {
```

Comment:
As a follow-up to this PR, we should probably combine `make_key_block` and `new_tls12` to make it clearer than the key block is the right size.